### PR TITLE
Show progress bar while downloading bugs

### LINKED ISSUE #134: Print progress bar while downloading bugs
Similar to what we're doing for commits data: 28b83c12c29185c52afb58d94a533a9448969a8a.

### PR DESCRIPTION
**Pull request by ayir**
Fixes #134.

### CODE REVIEW

File: bugbug/bugzilla.py [hunk 1]
```diff
@@ -137,12 +137,14 @@ def download_bugs(bug_ids, products=None, security=False):
     new_bug_ids = sorted(list(new_bug_ids))
 
     total_downloaded = 0
+    pbar = tqdm(total=len(new_bug_ids))
```
Comment (marco-c) [inline bugbug/bugzilla.py]: You are not closing the `pbar`.
Maybe use `with`, so you can avoid closing it manually.

File: bugbug/bugzilla.py [hunk 2]
```diff
@@ -137,12 +137,14 @@ def download_bugs(bug_ids, products=None, security=False):
...
     chunks = (new_bug_ids[i:(i + 500)] for i in range(0, len(new_bug_ids), 500))
+
     for chunk in chunks:
         new_bugs = _download(chunk)
 
         total_downloaded += len(new_bugs)
-
+        pbar.update(total_downloaded)
```
Comment (marco-c) [inline bugbug/bugzilla.py]: I guess you haven't actually tested this?
The parameter to `update` is not the current total, but the step.

Comment (ayir) [reply bugbug/bugzilla.py]: Hey I modified the code and updated by length of chunk.
Is it fine now?

**Comment by marco-c** [comment]
It looks better, thanks!

File: bugbug/bugzilla.py [hunk 3]
```diff
@@ -5,12 +5,12 @@
 
 import json
 import os
-
+import time
```
Comment (marco-c) [inline bugbug/bugzilla.py]: This is not needed, can you remove it?

File: bugbug/bugzilla.py [hunk 4]
```diff
@@ -5,12 +5,12 @@
...
 import requests
 from libmozdata import bugzilla
 
 from bugbug import db
-
+from tqdm import tqdm
 BUGS_DB = 'data/bugs.json'
```
Comment (marco-c) [inline bugbug/bugzilla.py]: There's probably going to be a flake8 error here, as you removed the newline after the imports.

File: bugbug/bugzilla.py [hunk 5]
```diff
@@ -135,20 +135,22 @@ def download_bugs(bug_ids, products=None, security=False):
     print(f'Loaded {old_bug_count} bugs.')
 
     new_bug_ids = sorted(list(new_bug_ids))
-
```
Comment (marco-c) [inline bugbug/bugzilla.py]: Don't remove this newline

File: bugbug/bugzilla.py [hunk 6]
```diff
@@ -135,20 +135,22 @@ def download_bugs(bug_ids, products=None, security=False):
...
     total_downloaded = 0
-    chunks = (new_bug_ids[i:(i + 500)] for i in range(0, len(new_bug_ids), 500))
-    for chunk in chunks:
-        new_bugs = _download(chunk)
 
-        total_downloaded += len(new_bugs)
+    chunks = (new_bug_ids[i:(i + 500)] for i in range(0, len(new_bug_ids), 500))
+    with tqdm(total=len(new_bug_ids)) as pbar:
```
Comment (marco-c) [inline bugbug/bugzilla.py]: Nit: change `pbar` to `progress_bar`, as it's more readable

File: bugbug/bugzilla.py [hunk 7]
```diff
@@ -135,20 +135,22 @@ def download_bugs(bug_ids, products=None, security=False):
...
+        for chunk in chunks:
+            new_bugs = _download(chunk)
 
-        print(f'Downloaded {total_downloaded} out of {len(new_bug_ids)} bugs')
+            total_downloaded += len(new_bugs)
+            time.sleep(0.1)
+            pbar.update(len(chunk))
+            # print(f'Downloaded {total_downloaded} out of {len(new_bug_ids)} bugs')
```
Comment (marco-c) [inline bugbug/bugzilla.py]: You can now remove this line altogether

File: bugbug/bugzilla.py [hunk 8]
```diff
@@ -135,20 +135,22 @@ def download_bugs(bug_ids, products=None, security=False):
...
-        new_bugs = _download(chunk)
 
-        total_downloaded += len(new_bugs)
+    chunks = (new_bug_ids[i:(i + 500)] for i in range(0, len(new_bug_ids), 500))
+    with tqdm(total=len(new_bug_ids)) as pbar:
+        for chunk in chunks:
+            new_bugs = _download(chunk)
 
-        print(f'Downloaded {total_downloaded} out of {len(new_bug_ids)} bugs')
+            total_downloaded += len(new_bugs)
```
Comment (marco-c) [inline bugbug/bugzilla.py]: We no longer need `total_downloaded`, as the progress bar is going to show the progress

File: bugbug/bugzilla.py [hunk 9]
```diff
@@ -5,11 +5,11 @@
 
 import json
 import os
-
```
Comment (marco-c) [inline bugbug/bugzilla.py]: This newline should not be removed

**Comment by marco-c** [comment]
Ah, it looks like there is still a linting error in the build.
`./bugbug/bugzilla.py:13:1: I001 isort found an import in the wrong position`

You can reproduce locally by running `flake8`.

**Pull request merged**